tmt/steps/discover/fmf.py

```diff
@@ -155,6 +155,12 @@ class DiscoverFmfStepData(tmt.steps.discover.DiscoverStepData):
         show_default=True,
         help="Copy only immediate directories of executed tests and their required files.")
 
+    # Edit discovered tests
+    adjust_tests: list[Any] = field(
```

Comment:
Would it make sense for this field to be pretty much the same as `adjust`, https://github.com/teemtee/tmt/blob/main/tmt/base.py#L681 ?

Reply:
I suppose it could, it would be re-using the `adjust` concept, i.e. the same set of keys? We would use `_RawAdjustRule` type instead of `Any` (it's almost the same, but still better than `Any`, it's easier to follow in code).

Reply:
thanks, was blind to see that we already have adjust rules